e2e/playwright.config.ts

```diff
@@ -0,0 +1,37 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+require('dotenv').config();
+
+const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  /* Maximum time one test can run for. */
+  timeout: 120 * 1000,
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 20000,
+  },
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  globalSetup: require.resolve('./global-setup'),
+  use: {
+    /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
+    actionTimeout: 0,
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    // baseURL: 'http://localhost:3000',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+  },
```

Comment:
Do you want to run the tests for the different browsers too? My thinking is:
- GSIB users probably are on MS Edge (or maybe Chrome)
- OGP users are probably on Chromium and Webkit (e.g. I know Kenneth runs email campaigns on his phone)

This is the config from CheckWho:
```typescript
projects: [
    {
      name: 'chromium',
      use: {
        ...devices['Desktop Chrome'],
      },
    },
    {
      name: 'firefox',
      use: {
        ...devices['Desktop Firefox'],
      },
    },

    {
      name: 'webkit',
      use: {
        ...devices['Desktop Safari'],
      },
    },

    /* Test against mobile viewports. */
    {
      name: 'Mobile Chrome',
      use: {
        ...devices['Pixel 5'],
      },
    },
    {
      name: 'Mobile Safari',
      use: {
        ...devices['iPhone 12'],
      },
    },

    /* Test against branded browsers. */
    {
      name: 'Microsoft Edge', // browser used by public officers
      use: {
        channel: 'msedge',
      },
    },
  ],
```

Reply:
I don't think these browser compat is big enough an issue to revert a deployment, so not intending to test on lower-tier ones

Reply:
so which are the browsers we are actually testing on? I actually think we should prioritise Edge + Chrome on PC (since that's what our users use)

Reply:
The default one is Chromium